src/less/pagination/ds6/pagination.less

```diff
@@ -1,4 +1,129 @@
 @import "../../less/ds6/variables.less";
 @import "../../less/ds6/mixins.less";
 
-@import "pagination-base.less";
+@import "../base/pagination.less";
+
+.pagination a {
```

Comment:
Instead of having color on the link, should we have it on the class?

Reply:
This is a carry-over from somewhere. Let me look into why I added it.